modules/azure/src/test/java/org/testcontainers/containers/CosmosDBEmulatorContainerTest.java

```diff
@@ -0,0 +1,45 @@
+package org.testcontainers.containers;
+
+import com.azure.cosmos.CosmosAsyncClient;
+import com.azure.cosmos.CosmosClientBuilder;
+import com.azure.cosmos.models.CosmosContainerResponse;
+import com.azure.cosmos.models.CosmosDatabaseResponse;
+import org.junit.Rule;
+import org.junit.Test;
+import org.testcontainers.utility.DockerImageName;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * @author Onur Kagan Ozcan
+ */
+public class CosmosDBEmulatorContainerTest {
+
+    @Rule
+    // emulatorContainer {
+    public CosmosDBEmulatorContainer emulator = new CosmosDBEmulatorContainer(
+        DockerImageName.parse("mcr.microsoft.com/cosmosdb/linux/azure-cosmos-emulator")
+    );
+    // }
+
+    @Test
+    public void testSimple() {
+    // buildClient {
+        CosmosAsyncClient client = new CosmosClientBuilder()
+            .gatewayMode()
+            .endpointDiscoveryEnabled(false)
+            .endpoint(emulator.getEmulatorEndpoint())
+            .key(CosmosDBEmulatorContainer.EMULATOR_KEY)
```

Comment:
Here I wonder if `EMULATOR_KEY` should be a public constant, or if the API should be more like:
```
emulator.getKey();
```

This of course could internally just return the constant. Not sure if this is more idiomatic, but this API seems more symmetrical to how we implement it in other database modules and feels like a better encapsulation to me.

Reply:
Actually getter is better IMO, aforetime I implemented it as getter and changed to constant by your request.

Reply:
You are totally right, sorry for that. I think with my changed understanding of the PR, I would indeed prefer the getter.